src/components/popup/popup.md

```diff
@@ -0,0 +1,9 @@
+The Popup component can be used to show something under a conditional of any sort:
```

Comment:
This is an indication that this is _not_ the right abstraction. If the purpose of the component is conditional  rendering then why should we use this instead of the normal React practice of `someBool && <TheComponent />`?